.travis.yml

```diff
@@ -12,9 +12,24 @@ install:
   # Install extra requirements for running tests and building docs.
   - pip install -r requirements-dev.txt
 
+env:
+   global:
+    # Doctr deploy key for NSLS-II-SIX/sixtools
+    - secure: "UG3NQz4KFsIdYsHhqpbeImJczqWMm6TsrOi6HXvdM+v93RUZW87J/ahZ537pEU3WF1pF3IK0yLbCz7angLf0o2r2KDP2h70h7mx5v1o2hXyhqHibcDkkAg2p9J28xQ9yiZHohS/ARRReBAMjf5g1LPVQ20pxCEDpCX7+HXgN9WqOIKbugvqF4o1len6U3lFlduAiukM8rkfdNs+lSn8fLUlCGLdUOw5e45P66mABZ8dpcP36bZUAdCG0wp/VaXPHWaGog5tA+/l75HjGFhJd7CragCXy9Lfo9MtrimPJmprUiQ46ckBIQzUDtLMygBA9/p+ywwqphUoidCqDYNECWUm2TDBbUpSrEG7CPTnbLnQRTPTtunVWe8RDOO6GvQXalAUJSfd8C2MeJ9qKp5Kb4q6v/L2uvCYN+0CJ6+ex85UGJgQKctBB/blU9PlofN4p67TDsp9rs0AkkgQ3OXx/+CS6AfVFkhEPFhF15MxZHVnj8A9S5Hb6Qc2smFiyiIWoDiOG/prZA/VRBI4lcFJ0et5qNTJ+LdFNnURcXtyV/dVJthas6+ceK+kjA3oOEtveqys1imLkvx/u4kGBwzDO5yhObsD8SuXsWd3UJFXW3OSAkZoAYQM6sVio56Kd0TiqhWGLYixQgbHBxQa5Nuw8kd+X0Cb4gafmmC6LN8XISVE="
+
 script:
   - coverage run -m pytest  # Run the tests and check for test coverage.
   - coverage report -m  # Generate test coverage report.
   - codecov  # Upload the report to codecov.
   - flake8 --max-line-length=115  # Enforce code style (but relax line length limit a bit).
-  - make -C docs html  # Build the documentation.
+  - set -e
+  #build docs
+  - |
+    if [ $BUILD_DOCS]; then
+        pushd doc
+        make html
+        popd
+        #publish docs
+        doctr deploy --deploy-repo NSLS-II-SIX/NSLS-II-SIX.github.io --deploy-branch-name master sixtools
```

Comment:
This `--deploy-repo NSLS-II-SIX/NSLS-II-SIX.github.io --deploy-branch-name master` customization (incidentally, [added by me -- see PR for context on what this is doing](https://github.com/drdoctr/doctr/pull/190)) isn't necessary unless we want to go the route of one centralized target for built artifacts. We do that for NSLS-II/* repos, but it _probably_ is overkills for SIX.